mimic/rest/maas_api.py

```diff
@@ -337,7 +338,7 @@ def create_entity(self, request, tenant_id):
         """
         postdata = json.loads(request.content.read())
         myhostname_and_port = 'http://' + request.getRequestHostname() + ':' + self.endpoint_port
-        newentity = createEntity({'label': postdata[u'label'].encode('ascii')})
+        newentity = create_entity(postdata)
```

Comment:
This is the major change. Earlier `create_entity` just passed along the entity name and no other attributes from the request payload.